### ISSUE TITLE
ESIPP [Slow] e2e tests failing on 5000 node cluster jobs

### ISSUE THREAD

**Issue by porridge** [kubernetes/kubernetes#56138]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

This is a followup from #52495 filed separately now that we're not tracking this as part of 1.9 release. The tests have been disabled with [DisabledForLargeClusters] tag for now, and this fact will be mentioned in the scalability validation report for v1.9.

This issue tracks fixing the test so the tag can be removed and the test ran again in the large cluster.

/assign @MrHohn 
@kubernetes/sig-network-bugs 
@kubernetes/sig-scalability-bugs 
/kind e2e-test-failure

**Comment by jberkus** [comment]
Is this e2e test still broken?

**Comment by MrHohn** [comment]
@jberkus Sorry for the confusion here. Currently this e2e test is disabled on extra large cluster due to some scalability issues with the load balancer related components. At the moment the issues haven't been completely resolved yet.

**Comment by jberkus** [comment]
ok, adding priority so that I know it's not a 1.13 blocker.

/priority important-longterm

**Comment by yastij** [comment]
cc @lbernail - this is why the relevant tests for #72432 are not part of the e2e suite.

**Comment by MrHohn** [comment]
This issue is still valid --- some action items on top of my head:
- Figure out a plan to make service controller properly handle operations in parallel.
- Check on the status for cloud providers (especially GCP, which the test ran against) on handling large number of nodes as LB backends.

/remove-triage unresolved

**Comment by cblecker** [comment]
/reopen
Reopening as #89421 is being reverted in #90171

**Comment by liggitt** [comment]
these tests are flaking in https://testgrid.k8s.io/sig-release-master-informing#gce-master-scale-correctness&width=20

xref https://github.com/kubernetes/kubernetes/issues/90955

**Comment by liggitt** [comment]
cc @wojtek-t, since these were enabled by #90305

**Comment by wojtek-t** [comment]
Yeah - though these failures doesn't seem to be scale related, they seem to be test related to me. 
This basically means nodes without public ips:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/service.go#L2928

It's pretty much the same thing as https://github.com/kubernetes/kubernetes/issues/90047 (just for a different test).
https://github.com/kubernetes/kubernetes/pull/92163 should be addressing these kinds of errors.

[Not a release blocker for sure]

**Comment by wojtek-t** [comment]
BTW - https://github.com/kubernetes/kubernetes/issues/93341 has been opened directly for this one.

**Comment by jayunit100** [comment]
@wojtek-t  ... so since #93341 is closed, Can we close this and re-enable for large clusters ?   (i.e. remove the Tag in service.go that disables this test) ? I'm not up to speed on the history but 2 related notes I came across digging into related test `ESIPP [Slow] should handle updates to ExternalTrafficPolicy field` today... 

- There's a TODO in the code referencing this issue, but the direct issue https://github.com/kubernetes/kubernetes/issues/93341 is now closed, so we can close this as well right?... And if so, we can remove the  TODO on the service.go test...

```
	// TODO: Get rid of [DisabledForLargeClusters] tag when issue #56138 is fixed.
	// [LinuxOnly]: Windows does not support session affinity.
	ginkgo.It("should have session affinity work for LoadBalancer service with ESIPP off [Slow] [DisabledForLargeClusters] [LinuxOnly]", func() {
```
- enabling the test ... not sure wether the timing is good right now (given  that https://github.com/kubernetes/kubernetes/issues/93932 is open right now which is a sorta related test)

**Comment by wojtek-t** [comment]
We tried to enable all of them and that caused us a bunch of issues and those had to be reverted:
https://github.com/kubernetes/kubernetes/pull/90171

A couple tests were enabled later in: https://github.com/kubernetes/kubernetes/pull/90305 and those seem fine.

The remaining (mostly session-affinity ones) had some issue, which so far we didn't have time to debug and I haven't heard a good explanation why those were failing.
It's possible that they were improved with #92427, but I doubt those would really cause the failures we've seen in the past.


So until this is debugged (or at least understood) I'm reluctant to enabling them.
